std/assembly/rt/itcms.ts

```diff
@@ -194,7 +199,7 @@ function step(): usize {
         iter = obj.next;
         if (DEBUG) assert(obj.color == i32(!white)); // old white
         free(obj);
-        return 1;
+        return 10;
```

Comment:
why `10`? What does this value means?

Reply:
This is a magic number introduced by means of guessing, representing the overall "cost" attached to sweeping one object. Affects how many steps the GC performs in succession when it runs incrementally. Per run it has a budget of `GRANULARITY * EAGERNESS / 100` (=400 by default).

Reply:
Could it be hoist to some named enum of constant?)